webpack.common.js

```diff
@@ -56,13 +56,15 @@ const targets /*: Array<Target> */ = [
  */
 function createConfig(target /*: Target */, dev /*: boolean */,
         minimize /*: boolean */) /*: Object */ {
-    const cssLoaders /*: Array<Object> */ = [{loader: 'css-loader'}];
+    const cssLoaders /*: Array<Object> */ = [{
+        loader: 'css-loader',
+        options: {importLoaders: 1},
+    }, {
+        loader: 'postcss-loader',
+        options: {plugins: [require('postcss-preset-env')()]},
```

Comment:
I assume this reads our .browserslistrc.

Reply:
@kevinbarabash Yes, it does.